pycbc/events/ranking.py

```diff
@@ -4,6 +4,49 @@
 import numpy
 
 
+def newsnr_sgveto_psdvar_scaled(snr, bchisq, sgchisq, psd_var_val):
+    """ Combined SNR derived from NewSNR, Sine-Gaussian Chisq and PSD
+    variation statistic """
+    nsnr = numpy.array(newsnr_sgveto(snr, bchisq, sgchisq), ndmin=1)
+    psd_var_val = numpy.array(psd_var_val, ndmin=1)
+    # 1.2 is the expected maximum psd_var_val over gaussian noise.
+    nsnr = nsnr / (psd_var_val)**0.33
```

Comment:
Both the 1.2 and 0.33 numbers

Reply:
0.33 should be set via kwarg I think with a comment that it was empirically tuned on O2 triggers

Reply:
Also pep8 please